bin/cli-flags.js

```diff
@@ -88,7 +89,8 @@ module.exports = {
       type: Boolean,
```

Comment:
I think we no longer need these types, lets remove

Reply:
I'm afraid we will lose compatibility with the webpack-cli version <= 4.5.0

Reply:
yep lets keep it